app/modbus.mjs

```diff
@@ -308,13 +331,17 @@ export const getDeviceFamilyName = (familyTypeInt) => {
             'Pegasos',
             'Pegasos XL',
             'LTR-3',
-            'LTR-6̈́',
+            'LTR-6',
             'LTR-7',
             'LTR-7 XL',
         ][familyTypeInt] || 'unknown'
     )
 }
 
+export const getDeviceProName = (proTypeInt) => {
+    return ['RS', 'RSC', 'LTR', 'LTC', 'LTT', 'LTP'][proTypeInt] || 'unknown'
```

Comment:
You can add a comment after the first item (see `getDeviceFamilyName`) to force `prettier` to not format like this. It's dumb, I know.